README.md

```diff
@@ -36,8 +36,26 @@ repos.
 
 ## Usage
 
-TODO: fill in details and examples here.
+### Commitlint
 
+* Add a file called commitlint.config.js to your project root with the following content:
```

Comment:
Could you run markdownlint with our standards against this file? I think these bullet points should be indented, and that would help catch any other errors.

Reply:
I was waiting to lint the markdown file until the markdown lint config is merged. 🐔 or 🥚